CONTRIBUTING.md

```diff
@@ -75,19 +72,18 @@ Below is a short list of guiding rules that you should keep in mind when you are
 - **DO** use the [template](./styleguide/template.md) file as the starting point of your work.
 - **DO** create a separate branch on your fork before working on the articles.
 - **DO** follow the [GitHub Flow workflow](https://guides.github.com/introduction/flow/). 
-- **DO** blog and tweet (or whatever) about your contributions, frequently!
+- **DO** blog, tweet and use other social media to annouce your contribution.
 
 > Note: you might notice that some of the topics are not currently following all the guidelines specified here and on the [style guide](./styleguide/template.md) as well. We're working towards achieving consistency throughout the site. Check the list of [open issues](https://github.com/dotnet/core-docs/issues?q=is%3Aissue+is%3Aopen+label%3Aguidelines-adherence) we're currently tracking for that specific goal. 
 
 ## Building the docs
 
 The documentation is written in [GitHub Flavored Markdown](https://help.github.com/categories/writing-on-github/) and built using [DocFX](http://dotnet.github.io/docfx/) and other internal publishing/building tools. It is hosted at [docs.microsoft.com](https://docs.microsoft.com/dotnet). 
 
-To build the docs locally, you need to install [DocFX](https://dotnet.github.io/docfx/); latest versions are the best.
+Install [DocFX](https://dotnet.github.io/docfx/) to build the docs.
```

Comment:
there are two types of builds: local and automatically triggered OPS builds. not sure how locally can be confusing.